app/views/patients/index.html.erb

```diff
@@ -13,7 +13,7 @@
   <div class="card card-square">
     <h5 class="card-header">Enrolled Monitorees</h5>
     <div class="m-3">
-      <%= render partial: 'patients', locals: { patients: current_user.enrolled_patients } %>
+      <%= render partial: 'patients', locals: { patients: current_user.enrolled_patients.eager_load(:jurisdiction) } %>
```

Comment:
Any reason in particular for `eager_load` over `includes`?

Reply:
I chose to do `eager_load` because it produces a single database query and `includes` produces two queries (one for patients and one for jurisdictions). Both have very similar performance in the local development server, so considering that I figure it would be better to do a single query over two queries.